Sources/Property.swift

```diff
@@ -678,3 +759,73 @@ public final class MutableProperty<Value>: MutablePropertyProtocol {
 		observer.sendCompleted()
 	}
 }
+
+extension MutableProperty {
```

Comment:
It could, yes. But it's not really holding up the behavior right now.

If #182 goes in first, I'll definitely merge & apply to that. If #182 continues to take a while, I'm OK with a separate PR that moves this to that protocol.